community/modules/scripts/ramble-execute/templates/ramble_execute.yml.tpl

```diff
@@ -28,20 +28,18 @@
 
     - name: Execute ramble commands
       ansible.builtin.shell: |
-        set -e
         . {{ spack_path }}/share/spack/setup-env.sh
         . {{ ramble_path }}/share/ramble/setup-env.sh
+
+        set -eo pipefail
         echo " === Starting ramble commands ==="
+        {
         {{ commands }}
+        } | tee -a {{ log_file }}
```

Comment:
nit: I would wrap everything in the tee, including the setup-env and the echo commands.